src/factors/Range2D.jl

```diff
@@ -9,7 +9,7 @@ mutable struct Point2Point2Range{D <: IIF.SamplableBelief} <: IncrementalInferen
 end
 
 function getSample(cfo::CalcFactor{<:Point2Point2Range}, N::Int=1)
-  return (reshape(rand(cfo.factor.Z,N),1,N),)
+  return (rand(cfo.factor.Z, N),)
```

Comment:
vector of tangents?

Reply:
It doesn't seem to matter, will keep like this as a test for now as it should not matter as samples are a Vector{P} and P can be a Float